src/Sylius/Bundle/AdminBundle/Resources/views/Product/Grid/Action/updatePositions.html.twig

```diff
@@ -1,4 +1,6 @@
-<form class="sylius-update-product-taxons" action="{{ path('sylius_admin_ajax_product_taxons_update_position') }}" data-csrf-token="{{ csrf_token('update-product-taxon-position') }}">
+<form id="sylius-update-product-taxons" class="sylius-update-product-taxons" action="{{ path('sylius_admin_ajax_product_taxons_update_position') }}" method="post">
```

Comment:
Is it still an ajax call? We cannot remove this route, but maybe depreciation and adding a new one would be a good option?